flytekit/models/core/identifier.py

```diff
@@ -33,6 +33,9 @@ def resource_type(self):
         """
         return self._resource_type
 
+    def resource_type_name(self) -> str:
```

Comment:
can we add a unit test for this?

Reply:
it all comes from generated protobuf code but sure